catroid/src/main/java/org/catrobat/catroid/content/Script.java

```diff
@@ -146,6 +146,12 @@ public void removeInstancesOfUserBrick(UserBrick userBrickToRemove) {
 		}
 	}
 
+	public void removeBricksWithInvalidReferences(List<Brick> bricksWithInvalidReferences) {
```

Comment:
Please rename to `removeBricks(List<Brick> bricks)`. So it's a more generic method.